sdk/ml/azure-ai-ml/tests/internal/e2etests/test_pipeline_job.py

```diff
@@ -306,3 +306,24 @@ def training_pipeline_with_components_in_registry(input_data, test_data, learnin
         )
         pipeline_job.settings.default_compute = "cpu-cluster"
         assert_job_cancel(pipeline_job, client, experiment_name="v15_v2_interop")
+
+    def test_internal_component_node_output_type(self, client):
+        from azure.ai.ml._utils.utils import try_enable_internal_components
+
+        # force register internal components after partially reload schema files
+        try_enable_internal_components(force=True)
+        yaml_path = "./tests/test_configs/internal/command-component-single-file-output/component_spec.yaml"
+        component_func = load_component(yaml_path)
+
+        @pipeline
+        def pipeline_func():
+            node = component_func()
+            # node level should not have output type when type not configured
```

Comment:
should have correct type instead of not ?

Reply:
Will fix in next PR